core/vm/actor/builtin/v2/miner/miner_actor.hpp

```diff
@@ -11,8 +11,18 @@ namespace fc::vm::actor::builtin::v2::miner {
   using primitives::TokenAmount;
 
   using Construct = v0::miner::Construct;
+  using ControlAddresses = v0::miner::ControlAddresses;
+  using ChangeWorkerAddress = v0::miner::ChangeWorkerAddress;
+  using ChangePeerId = v0::miner::ChangePeerId;
+  using SubmitWindowedPoSt = v0::miner::SubmitWindowedPoSt;
+  using PreCommitSector = v0::miner::PreCommitSector;
+  using ProveCommitSector = v0::miner::ProveCommitSector;
+  using ExtendSectorExpiration = v0::miner::ExtendSectorExpiration;
+  using TerminateSectors = v0::miner::TerminateSectors;
+  using DeclareFaults = v0::miner::DeclareFaults;
+  using DeclareFaultsRecovered = v0::miner::DeclareFaultsRecovered;
   using OnDeferredCronEvent = v0::miner::OnDeferredCronEvent;
-  using ConfirmSectorProofsValid = v0::miner::ConfirmSectorProofsValid;
+  using CheckSectorProven = v0::miner::CheckSectorProven;
 
   struct ApplyRewards : ActorMethodBase<14> {
```

Comment:
Do we have no AddLockedFund method in v2?

Reply:
No, it was renamed to `ApplyRewards`.

Reply:
ok